### DISCUSSION TITLE
flickr icon

### DISCUSSION THREAD

**Discussion by CaroWebdesign** [twbs/bootstrap#40918] [Ideas]
Hello everybody,

a customers use flickr but i can't set it in the website with bootstrap-icons only.
Can you add the flickr icon please?

**Comment by julien-deramond** [comment]
Hello @CaroWebdesign 
FYI, there's an icon request at https://github.com/twbs/icons/issues/1505 for Flickr.

**Comment by aniruddhaadak80** [comment]
Hi @CaroWebdesign ,  

Currently, Bootstrap Icons does not include a Flickr icon. You might consider using an alternative icon library like Font Awesome or SVG icons for the Flickr icon. You can also create a custom SVG icon and integrate it into your project.  

Thanks!
